diff -urN a/BUILD.bazel b/BUILD.bazel
--- a/BUILD.bazel	2021-01-27 15:40:47.000000000 -0500
+++ b/BUILD.bazel	2021-01-27 15:41:18.000000000 -0500
@@ -8,7 +8,7 @@
         "middleware.go",
     ],
     importpath = "github.com/aws/aws-sdk-go-v2/service/internal/presigned-url",
-    visibility = ["//:__subpackages__"],
+    visibility = ["//visibility:public"],
     deps = [
         "@com_github_aws_aws_sdk_go_v2//aws/middleware:go_default_library",
         "@com_github_aws_aws_sdk_go_v2//aws/signer/v4:go_default_library",
